Sources/tuist/main.swift

```diff
@@ -1,5 +1,13 @@
 import Foundation
-import TuistKit
+import enum Basic.ProcessEnv
+
+if CommandLine.arguments.contains("--verbose") {
+    try? ProcessEnv.setVar("TUIST_VERBOSE", value: "true")
+}
 
+import enum TuistSupport.LogOutput
```

Comment:
Nitpick What about moving this to the top for consistency with how we define the imports.